src/pages/Feedback/Feedback.jsx

```diff
@@ -1,54 +1,69 @@
 import React, { useState, useEffect } from 'react'
-import { Row, Nav } from 'react-bootstrap'
-import Page from '@/components/Page/Page'
+import { Row, Container } from 'react-bootstrap'
 import FeedbackForm from '@/forms/Feedback'
 import FeedbackApi from '@/API/feedback'
 import Item from './components/Item'
+import ContentBox from '@/components/ContentBox/ContentBox'
+import alert from '@/utils/alert'
+
+function getRecipientObj(recipient, recipients) {
+    const obj = recipients.find((rec) => rec.name === recipient)
+    return obj
+}
 
 export default function Feedback() {
     const [recipients, setRecipients] = useState([])
-    const [recipient, setRecipient] = useState({})
+    const [recipient, setRecipient] = useState('')
 
     useEffect(() => {
         async function FetchApi() {
             const response = await FeedbackApi.getAll()
             setRecipients(response?.data)
-            setRecipient(response?.data[0])
+            setRecipient(response?.data[0].name)
         }
         FetchApi()
     }, [])
 
-    function sendRequest(data) {
-        FeedbackApi.post(recipient.name, data)
+    async function sendRequest(values) {
+        const data = { category: values.category, text: values.text }
+        if (values.email) data.email = values.email
+
+        if (await FeedbackApi.post(getRecipientObj(recipient, recipients)?.name, data)) {
+            alert('Вы успешно отправили свой запрос', {
+                title: 'Успех!',
+            })
+        }
     }
 
     return (
-        <>
-            <Page header="Шаг 1. Укажите, кому будет направлен ваш вопрос">
-                <Nav
-                    variant="pills"
-                    className="my-3"
-                    activeKey={recipient?.name}
-                    onSelect={(eventKey) => {
-                        setRecipient(recipients.find((rec) => rec?.name === eventKey))
-                    }}
+        <Container className="my-5">
```

Comment:
Если хочется сыимитировать поведение обычного пейджа, то давай так:
```suggestion
        <Container className="mt-4 mb-5">
```